runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -201,14 +207,17 @@ private boolean isTableTagged(String tableBackup) {
             rt.getTableRegistry()
                     .getRegistryTable()
                     .entryStream()
-                    .forEach(entry -> {
+                    .forEachOrdered(entry -> {
```

Comment:
Since these are processed one-by-one, why not use `sequential()` to avoid the overheard of coordination between threads? In this manner, the parallel stream provides no benefits.

Reply:
Done. Thanks.